iroha-cli/interactive/impl/interactive_query_cli.cpp

```diff
@@ -167,6 +172,22 @@ namespace iroha_cli {
           local_time_, creator_, counter_, account_id);
     }
 
+    std::shared_ptr<iroha::model::Query>
+    InteractiveQueryCli::parseGetTransactions(QueryParams params) {
+      // TODO 22/11/17 motxx - Parser grammer is obvious for user.
```

Comment:
What do you mean here? make it obvious?

Reply:
please mark TODOs in the way that they contain link to GitHub issue or JIRA task

Reply:
I removed todo, no problem for now.